main.c

```diff
@@ -43,6 +43,7 @@ int main(int argc, char *argv[])
 
   // Create Cartesian process topology
   MPI_Init(&argc, &argv);
+  jl_eval_string("import MPI; MPI.run_init_hooks()");
```

Comment:
@vchuravy , does this look right to you? It works like that on Daint, but there the MPI implementation is not OpenMPI based.

Reply:
Yeah that looks good.